src/components/shared/formItems/ProjectPayoutMods.tsx

```diff
@@ -329,10 +329,10 @@ export default function ProjectPayoutMods({
         : [...mods, newMod],
     )
 
-    if (handle) {
-      setSettingHandle(handle)
-      setSettingHandleIndex(editingModIndex)
-    }
+    // if (handle) {
```

Comment:
@mejango this fixes the duplicate project bug you showed me at 3:47 in your video. 

@peripheralist the third video I've attached shows the problem and the fix as a result of removing this block of code that sets the project handle in the state after the mod has been submitted. Do you know if there's any other reason it should be here?